src/main/c/src/SerialImp.c

```diff
@@ -3599,22 +3453,15 @@ JNIEXPORT jint JNICALL RXTXPort(readByte)( JNIEnv *env,
 	int timeout = get_java_var( env, jobj, "timeout", "I" );
 	/* char msg[80]; */
 
-/*
-	ENTER( "RXTXPort:readByte" );
-	report_time_start( );
-*/
 	bytes = read_byte_array( env, &jobj, fd, buffer, 1, timeout );
 	if( bytes < 0 ) {
-		LEAVE( "RXTXPort:readByte" );
 		throw_java_exception( env, IO_EXCEPTION, "readByte",
 			strerror( errno ) );
 		return -1;
 	}
 /*
-	LEAVE( "RXTXPort:readByte" );
 	sprintf( msg, "readByte return(%i)\n", bytes ? buffer[ 0 ] : -1 );
 	report( msg );
```

Comment:
I think these sprintf loggings could be removed, too. But one step after another.